os/kernel/preference/preference_read.c

```diff
@@ -31,8 +32,8 @@ static int preference_read_fs_key(char *path, preference_data_t *data)
 {
 	int fd;
 	int ret;
-	int read_size;
-	value_data_t value_data;
+	uint32_t check_crc;
```

Comment:
preference_read_fs_key is not good naming.
How about align with tizen's api?

Reply:
The name of internal functions are _read_data and _write_data in Tizen. I think it is different from our naming rules for internal functions. I will think more about function name.